src/main/java/com/guysfromusa/carsgame/v1/CarResource.java

```diff
@@ -84,4 +82,15 @@ public Long removeCar(@ApiParam(name = "name", value="Car name to be deleted") @
         return carService.deleteCarByName(name);
     }
 
+    @PostMapping(path= "{name}/fix")
+    @ApiOperation(value = "Repair crashed car", response = Car.class)
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Car successfully repaired"),
+        @ApiResponse(code = 400, message = "Car with given name does not exist")
+    })
+    public Car repairCar(@PathVariable("name") String name){
```

Comment:
should we return whole car after repair?

Reply:
I think http 200 should be enough

Reply:
maybe list of running games ;p